src/app/options.cpp

```diff
@@ -247,6 +315,13 @@ namespace
     constexpr const StringOption CONFIGURATION_OPTION = {"configuration"};
     constexpr const BoolOption PORTABLE_OPTION = {"portable"};
     constexpr const BoolOption RELATIVE_FASTRESUME = {"relative-fastresume"};
+    constexpr const StringOption SAVE_PATH_OPTION = {"save-path"};
+    constexpr const TriStateBoolOption PAUSED_OPTION = {"add-paused"};
+    constexpr const BoolOption SKIP_HASH_CHECK_OPTION = {"skip-hash-check"};
+    constexpr const StringOption CATEGORY_OPTION = {"category"};
+    constexpr const BoolOption SEQUENTIAL_OPTION = {"sequential"};
+    constexpr const BoolOption FIRST_AND_LAST_OPTION = {"first-and-last"};
```

Comment:
Maybe I have misunderstood the code or my comment was incomplete.
Why isn't it possible for user to use `--add-paused`, `--add-paused=true`, `--add-paused=false`?
In this case the omission of a value should be equal to `true`. And if the user doesn't use the command switch the application wide setting should apply.